### ISSUE TITLE
catch not firing when file not found

### ISSUE THREAD

**Issue by wayou** [philbooth/bfj#59]
```js
bfj.read(path, options)
  .then(data => {
    // :)
  })
  .catch(error => {
    // :(   <---------not fire when file not found
  });
```

I wonder how to handle this situation or just wrap bfj within another try catch.

**Comment by philbooth** [comment]
Oh, whoops. Thanks for reporting! I'll fix it over the weekend.

**Comment by philbooth** [comment]
Fixed in 9fe21ffc7f27edef8a4194ffe24a1476824ea577, published to npm in version `5.2.1`. Let me know if you're still having issues!
